Documentation/ApiOverview/Environment/Index.rst

```diff
@@ -0,0 +1,90 @@
+.. include:: ../../Includes.txt
+
+.. _Environment:
+
+===========
+Environment
+===========
+
+Since version 9.x the TYPO3 CMS core includes an environment class.
+This class contains all environment specific information, e.g. paths within the
+filesystem. This implementation replaces previously used global variables and
+constants like :php:`PATH_site`.
+
+The fully qualified class name is :php:`\TYPO3\CMS\Core\Core\Environment`. The
+class provides static methods to access the necessary information.
+
+Also for testing, the :php:`initialize()` method can be called to adjust the
+information.
+
+.. _Environment-project-path:
+
+project path
```

Comment:
Please use upper camel case or at least capitalization for headlines. Alternatively use the API method as headline.